server/src/api.ts

```diff
@@ -14,9 +14,13 @@ export interface GetPostResponse {
   post: Post;
 }
 
-// Comment APIs
-
 // Like APIs
+export type CreateLikeRequest = Like;
+export type CreateLikeResponse = {
+  message: string;
```

Comment:
no need for the backend to send these back. the client should expect a successful (2xx) request to have acted on the like, and returning the same object the client sent in the request isn't probably all that useful. I'd keep this an empty interface until we have something useful to add here.